src/code/dialogs/markerAddDialog.js

```diff
@@ -81,8 +81,7 @@ const MarkerAddDialog = WDialog.extend({
         const o = L.DomUtil.create("option", null, this._type);
         o.value = k;
         o.textContent = wX(k);
-        if (markers.has(k)) o.disabled = true;
-        else if (!defaultType) defaultType = k;
+        if (!defaultType) defaultType = k;
```

Comment:
`markers` shouldn't be used in the previous line too
then `defaultType` would always be defined